src/plugins/sync-plugin.js

```diff
@@ -194,14 +194,18 @@ const restoreRelativeSelection = (tr, relSel, binding) => {
     const anchor = relativePositionToAbsolutePosition(binding.doc, binding.type, relSel.anchor, binding.mapping)
     const head = relativePositionToAbsolutePosition(binding.doc, binding.type, relSel.head, binding.mapping)
     if (anchor !== null && head !== null) {
-      tr = tr.setSelection(TextSelection.create(tr.doc, anchor, head))
+      const selection = relSel.isNodeSelection
+        ? NodeSelection.create(tr.doc, Math.min(anchor, head))
```

Comment:
How does this behave when multiple nodes are selected?

Reply:
> How does this behave when multiple nodes are selected?

Hmmm good question! I'm actually not sure how to test this.

Reply:
According to the Prosemirror docs, a [Node Selection](https://prosemirror.net/docs/ref/#state.NodeSelection) _is a selection that points at a single node._ 

I believe a range that points to mulitple nodes or ranges is still considered a Text Selection, however, I don't know how or if it is possible to select multiple specific nodes at once.